src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsConformingFractures.cpp

```diff
@@ -661,7 +686,15 @@ void SinglePhasePoromechanicsConformingFractures::
         {
           // Compute local area contribution for each node
           array1d< real64 > nodalArea;
```

Comment:
I know this is not your doing but we should NEVER allocate lvArrays inside kernels. Can you add a big TODO in front of it.